editor/property_editor.cpp

```diff
@@ -1804,73 +1804,76 @@ CustomPropertyEditor::CustomPropertyEditor() {
-	add_child(text_edit);
+	value_vbox->add_child(text_edit);
 	text_edit->set_anchors_and_offsets_preset(Control::PRESET_WIDE, Control::PRESET_MODE_MINSIZE, 5);
+	text_edit->set_v_size_flags(Control::SIZE_EXPAND_FILL);
 	text_edit->set_offset(SIDE_BOTTOM, -30);
 
 	text_edit->hide();
 	text_edit->connect("text_changed", callable_mp(this, &CustomPropertyEditor::_text_edit_changed));
 
-	for (int i = 0; i < MAX_ACTION_BUTTONS; i++) {
-		action_buttons[i] = memnew(Button);
-		action_buttons[i]->hide();
-		add_child(action_buttons[i]);
-		Vector<Variant> binds;
-		binds.push_back(i);
-		action_buttons[i]->connect("pressed", callable_mp(this, &CustomPropertyEditor::_action_pressed), binds);
-	}
-
 	color_picker = nullptr;
 
 	file = memnew(EditorFileDialog);
-	add_child(file);
+	value_vbox->add_child(file);
 	file->hide();
 
 	file->connect("file_selected", callable_mp(this, &CustomPropertyEditor::_file_selected));
 	file->connect("dir_selected", callable_mp(this, &CustomPropertyEditor::_file_selected));
 
 	error = memnew(ConfirmationDialog);
 	error->set_title(TTR("Error!"));
-	add_child(error);
+	value_vbox->add_child(error);
 
 	scene_tree = memnew(SceneTreeDialog);
-	add_child(scene_tree);
+	value_vbox->add_child(scene_tree);
 	scene_tree->connect("selected", callable_mp(this, &CustomPropertyEditor::_node_path_selected));
 	scene_tree->get_scene_tree()->set_show_enabled_subscene(true);
 
 	texture_preview = memnew(TextureRect);
-	add_child(texture_preview);
+	value_vbox->add_child(texture_preview);
 	texture_preview->hide();
 
 	easing_draw = memnew(Control);
-	add_child(easing_draw);
+	value_vbox->add_child(easing_draw);
 	easing_draw->hide();
 	easing_draw->connect("draw", callable_mp(this, &CustomPropertyEditor::_draw_easing));
 	easing_draw->connect("gui_input", callable_mp(this, &CustomPropertyEditor::_drag_easing));
 	easing_draw->set_default_cursor_shape(Control::CURSOR_MOVE);
 
 	type_button = memnew(MenuButton);
-	add_child(type_button);
+	value_vbox->add_child(type_button);
 	type_button->hide();
 	type_button->get_popup()->connect("id_pressed", callable_mp(this, &CustomPropertyEditor::_type_create_selected));
 
 	menu = memnew(PopupMenu);
 	//	menu->set_pass_on_modal_close_click(false);
-	add_child(menu);
+	value_vbox->add_child(menu);
 	menu->connect("id_pressed", callable_mp(this, &CustomPropertyEditor::_menu_option));
 
 	evaluator = nullptr;
 
 	spinbox = memnew(SpinBox);
-	add_child(spinbox);
+	value_vbox->add_child(spinbox);
 	spinbox->set_anchors_and_offsets_preset(Control::PRESET_WIDE, Control::PRESET_MODE_MINSIZE, 5);
 	spinbox->connect("value_changed", callable_mp(this, &CustomPropertyEditor::_range_modified));
 
 	slider = memnew(HSlider);
-	add_child(slider);
+	value_vbox->add_child(slider);
 	slider->set_anchors_and_offsets_preset(Control::PRESET_WIDE, Control::PRESET_MODE_MINSIZE, 5);
 	slider->connect("value_changed", callable_mp(this, &CustomPropertyEditor::_range_modified));
 
+	action_hboxes = memnew(HBoxContainer);
```

Comment:
>So I hope I get Your blessing on this merge, @YeldhamDev?

```suggestion
	action_hboxes = memnew(HBoxContainer);
	action_hboxes->set_alignment(BoxContainer::ALIGN_CENTER);
```

Just add this and I can give you an approval. :wink: